package.json

```diff
@@ -11,22 +11,22 @@
   "scripts": {
     "test": "npm run eslint",
     "eslint": "eslint --ignore-pattern dist .",
-    "jest": "PORT=${PORT:-'8080'} BABEL_ENV=jest jest",
-    "dev-server": "PORT=${PORT:-'8080'} webpack-dev-server",
-    "http-server": "PORT=${PORT:-'8080'} http-server dist",
-    "build:react-redux": "NAME=react-redux webpack",
-    "build:reactive-react-redux": "NAME=reactive-react-redux webpack",
-    "build:react-tracked": "NAME=react-tracked webpack",
-    "build:constate": "NAME=constate webpack",
-    "build:unstated-next": "NAME=unstated-next webpack",
-    "build:zustand": "NAME=zustand webpack",
-    "build:react-sweet-state": "NAME=react-sweet-state webpack",
-    "build:storeon": "NAME=storeon webpack",
-    "build:react-hooks-global-state": "NAME=react-hooks-global-state webpack",
-    "build:use-context-selector": "NAME=use-context-selector webpack",
-    "build:mobx-react-lite": "NAME=mobx-react-lite webpack",
-    "build:use-subscription": "NAME=use-subscription webpack",
-    "build:salvoravida-react-redux": "NAME=salvoravida-react-redux webpack",
+    "jest": "cross-env PORT=8080 BABEL_ENV=jest jest",
```

Comment:
Does this allow changing `PORT` from CLI?

Reply:
i don't think so. anyway, i would like that you add cross-env, so i can merge future update without conflict